setup.py

```diff
@@ -148,5 +148,12 @@ def find_package_data(packages):
     license = 'BSD',
     package_data = package_data,
     include_package_data = True,
-    install_requires = ['NumPy==1.11.0', 'scipy', 'nose', 'hyperopt', 'scikit-learn'],
+    install_requires = [
+        'hyperopt',
+        'nose',
+        'NumPy==1.11.0',
+        'scikit-learn',
+        'scipy',
+        'xgboost',
```

Comment:
only real change here was adding `xgboost`. Aesthetic changes were organizing alphabetically and separating onto newlines